root/programs/C#/Frameworks/Infrastructure/CustomControl/Control/WebCustomTextBox.cs

```diff
@@ -326,12 +326,12 @@ public bool Validate(out string[] result)
             if (this.CheckType != null)
             {
                 // 必須入力チェック
-                if (this.CheckType.IsIndispensabile)
+                if (this.CheckType.IsIndispensable)
                 {
                     if ((text == ""))
```

Comment:
@yusukemaegawa 

I have modified the [above comments](https://github.com/OpenTouryoProject/OpenTouryo/pull/147#discussion_r51259469).

Please check.